### PR TITLE
[issue-60] Pravega txn API changes

### LINKED ISSUE #60: Pravega txn API changes
**Problem description**
In this commit:

https://github.com/pravega/pravega/commit/fc2791739e21fa49600d58dd5c43a06c79f6e78b

we changed the txn API to remove the various timeouts in the signature. For now, we have moved them to configuration, but we should expect some further changes.

**Problem location**
Sink connector.

**Suggestions for an improvement**
Change the use of txns according to the API changes.

### PR DESCRIPTION
**Pull request by EronWright**
**Change log description**
- move to Pravega 0.2.0-SNAPSHOT (741f7bb41f06e944e88022d5be58ebacb0d29bb5)
- update to use new txn APIs
- update .gitattributes file for gradle.properties

**Purpose of the change**
Compatibility with 0.2+
Closes #60 

**How to verify it**
Execute `FlinkExactlyOncePravegaWriterTest`

### CODE REVIEW

**Comment by EronWright** [comment]
Since this requires the use of a Pravega 0.2.0-SNAPSHOT, I'm merging it to a new working branch called `dev-0.2` for now.

File: .gitattributes [hunk 1]
```diff
@@ -7,6 +7,7 @@
 *.java text
 *.xml text
 *.sh text
+*.properties text
```
Comment (fpj) [inline .gitattributes]: As a suggestion, I prefer to have changes unrelated to the issue being resolved done in a separate PR. I won't block the PR on it, it is only a recommendation.
